app/app.rb

```diff
@@ -0,0 +1,86 @@
+ENV["RACK_ENV"] ||= "development"
```

Comment:
Likewise to rake, are there any good resources for understanding rack? I've got the basic idea, but still don't fully understand how it is working.